src/super_gradients/recipes/imagenet_vit_large.yaml

```diff
@@ -15,9 +15,9 @@
 defaults:
   - imagenet_vit_base
 
-
 dataset_params:
-  batch_size: 32
+  train_dataloader_params:
```

Comment:
This doesnt change the dataset_params which are actually used now in train_from_recipe. You are changing the recipe by doing this !

Reply:
You are right, i added batch_size again in b55399ddcb5c2994ef47ee02d68414563170817e